src/vc_util/src/issuer_api.rs

```diff
@@ -121,13 +121,8 @@ pub struct Icrc21ErrorInfo {
 
 #[derive(Clone, Debug, CandidType, Deserialize, Eq, PartialEq)]
 pub enum Icrc21Error {
-    #[serde(rename = "forbidden")]
```

Comment:
So far for the issuer we were following the convention from II's API, with lower_case_names_and_labels.  Is the use of CamelCase here mandated by ICRC-21?  I believe it would be good if we had only once convention throughout VC-issuer API.

Reply:
Yes, ICRC-21 mandates camel case. It does so to be consistent with ICRC-1, 2 & 3, which all use camel case variants. From the other officially specified interfaces, the [HTTP gateway protocol](https://internetcomputer.org/docs/current/references/http-gateway-protocol-spec#response-body-streaming-interface) also uses camel case, whereas the [management canister](https://internetcomputer.org/docs/current/references/ic-interface-spec#ic-candid) does not.

In general it seems to me, that newer interfaces are specified to use camel case. So I would argue, that we should actually use camel case in the issuer interface as well.

Reply:
Ok, we can change to CamelCase, but maybe let's do that in a separate PR for all of the issuer API, in coordination with API users. I.e. maybe let's keep lower_case for this PR?